components/Footer.js

```diff
@@ -1,85 +1,198 @@
 import IconTwitter from "./icons/Twitter"
 import IconGithub from "./icons/Github"
-import IconYoutube from "./icons/YouTube"
+import IconYoutubeGray from "./icons/YouTubeGray"
 import IconLinkedIn from "./icons/LinkedIn"
 import NewsletterSubscribe from "./NewsletterSubscribe"
 import Container from "./layout/Container"
 import Link from "next/link"
+import AsyncAPILogoLight from "./AsyncAPILogoLight"
+import IconSlack from "./icons/Slack"
+import IconTwitch from "./icons/Twitch"
+import Heading from "./typography/Heading"
 
 export default function Footer() {
   return (
-    <footer className="bg-gray-900 mt-12">
-      <div className="max-w-screen-xl mx-auto py-12 px-4 overflow-hidden sm:px-6 lg:px-8">
-        <nav className="-mx-5 -my-2 flex flex-wrap justify-center">
-          <ul className="flex flex-wrap justify-center">
-            <li className="px-5 py-2">
-              <Link href="/about">
-                <a className="text-base leading-6 text-gray-500 hover:text-white transition ease-in-out duration-300">About</a>
+    <footer className="bg-dark mt-12 margin: 0 auto">
+      <div className="max-w-screen-xl mx-auto py-4 sm:py-6 md:py-12 xl:py-16 px-3 overflow-hidden sm:px-6 lg:px-8 divide-y divide-cool-gray">
+        <nav className="-mx-5 -my-2 py-4 sm:py-10 px-6 mt-2 flex flex-wrap justify-between ">
+          <div className="mr-14 w-full md:w-auto">
+            <div className="">
+              <Link href="/">
+                <a className="cursor-pointer">
+                  <AsyncAPILogoLight className="h-13 w-auto mt-3" />
```

Comment:
```suggestion
                  <AsyncAPILogoLight className="h-10 w-auto mt-3" />
```

I think we should match the height of the logo in the footer with the height of the logo in the header